testgrid/tgrun/pkg/runner/embed/secondarynodecmd.sh

```diff
@@ -78,17 +102,18 @@ function wait_for_initprimary_done()
 }
 function main() 
 {
+  green "install jq for json processing"
+  install_jq
   green "wait for join command"
   secondaryJoin=$(wait_for_join_commandready)
   green "$secondaryJoin"
   green "run join command"
-  runJoinCommand "$secondaryJoin"
+  runJoinCommand
```

Comment:
i assume you are saving this for a subsequent commit. just noting it here so we dont forget. you will have to download the airgap bundle and disable internet similar to the runcmd.sh airgap is set [here](https://github.com/replicatedhq/kURL/blob/1100b3af5a2d0d16da9be413e52308e7ada32dfc/testgrid/tgrun/pkg/scheduler/types/types.go#L48)